### PR TITLE
deactivation support

### PR DESCRIPTION
**Pull request by gdestuynder**
This supports HRIS user deactivation
its missing LDAP users deactivation for users who are not staff (who are contributors)
the later could be a separate work item actually

### CODE REVIEW

**Comment by gdestuynder** [comment]
note: this currently has an issue i found during testing where it will deactivate certain accounts which arent staff and arent otherwise marked in some other way. discussing tomorrow.

**Comment by gdestuynder** [comment]
this includes a revert for #419

File: docs/PersonAPI.md [hunk 1]
```diff
@@ -129,7 +129,24 @@ Retrieve specific user profiles:
...
+- `/v2/users/`
+
+In this case you will receive a JSON document such as :
+```
+{
+  "Items": [profile_1, profile_2, ...],
+  "nextPage": None
+}
+```
+where `nextPage` will be set to a `user_id` if there are more pages to fetch.
```
Comment (fiji-flo) [inline docs/PersonAPI.md]: nextPage actually is an object:
```json
{ "id": "SOME_USER_ID" }
```
I fell for this and it drove me crazy to figure it out.

File: well-known-endpoint/tpl/mozilla-iam-publisher-rules [hunk 2]
```diff
@@ -2,10 +2,10 @@
   "create": {
     "uuid": [ "cis" ],
     "user_id": [ "ldap", "hris", "access_provider" ],
-    "primary_username": [ "ldap", "hris", "cis", "access_provider" ],
+    "primary_username": [ "cis" ],
```
Comment (fiji-flo) [inline well-known-endpoint/tpl/mozilla-iam-publisher-rules]: :+1:

File: docs/PersonAPI.md (hunk 1 above)
Comment (gdestuynder) [reply docs/PersonAPI.md]: ops didnt submit "review", i though i commented here
after some testing i seem to definitely be getting either none either a single user_id, not a structure

see the code also https://github.com/mozilla-iam/cis/pull/411/files#diff-3247471fbaab5e22352bc0bfc86e2063R244 this definitely gets all pages

**Comment by gdestuynder** [comment]
something's odd with travis testing, local tests dont have this error / doesnt seem code related

**Comment by gdestuynder** [comment]
non-squashed since we might need commit history here - in particular this reverts https://github.com/mozilla-iam/cis/pull/419 which we'll have to fix and pull back in

**Pull request merged**